Library/Formula/ice.rb

```diff
@@ -13,26 +10,19 @@ class Ice < Formula
     sha1 "7bdb347052b8df900c2c2e6797726a79d9c0242d" => :mountain_lion
   end
 
-  option 'doc', 'Install documentation'
-  option 'demo', 'Build demos'
+  option "without-java", "Compile without Java support."
+
+  depends_on :java => "1.7" if build.with? "java"
 
```

Comment:
The syntax here should be `depends_on :macos => :mavericks`, or `=> :yosemite` or such.